gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/EmergePluginExtension.kt

```diff
@@ -119,6 +119,13 @@ abstract class GitLabOptions {
 
 abstract class ProductOptions {
   abstract val tag: Property<String>
+
+  internal abstract val tagFromVariant: Property<Boolean>
```

Comment:
Internal so consumers can't access this directly, can only set through the `setFromVariant` extension, like `tag.setFromVariant()`